src/vcpkg/commands.dependinfo.cpp

```diff
@@ -210,17 +244,37 @@ namespace vcpkg::Commands::DependInfo
             return s;
         }
 
-        std::string create_graph_as_string(const std::set<std::string, std::less<>>& switches,
+        std::string create_mermaid_as_string(const std::vector<PackageDependInfo>& depend_info)
+        {
+            std::string s;
```

Comment:
I think it is good to add some comments, to describe the flow chart diagram format uses in this method

Reply:
I think the generated flow chart is explained perfectly well in the docs submission.